scheduler/launchd_test.go

```diff
@@ -0,0 +1,109 @@
+package scheduler
+
+import (
+	"os"
+	"testing"
+
+	"github.com/fluidkeys/fluidkeys/assert"
+)
+
+var ld = launchd{}
+
+func TestLaunchdEnable(t *testing.T) {
+	t.Run("enable successfully if file is present", func(t *testing.T) {
+		mockFileHelper := mockFileFunctions{
+			OsStatReturnError:          nil,
+			IoutilWriteFileReturnError: nil,
+		}
+		launchdEnabled, err := ld.enable(&mockFileHelper, "fake.plist")
+		assert.NoError(t, err)
+
+		assert.Equal(t, true, launchdEnabled)
+	})
+
+	t.Run("load successfully if file is missing", func(t *testing.T) {
```

Comment:
does "load successfully if file is missing" mean "create file if it's missing" ?